### ISSUE TITLE
Using FastText pretrained model to train new spacy model

### ISSUE THREAD

**Issue by anotherbugmaster** [explosion/spaCy#2051]
I use `python -m spacy train` script to train my model with parser, tagger and ner pipelines. I couldn't understand how spaCy obtains vectors for tokens and docs without me feeding them to it and I suspect that this is what causes poor quality of my model. To fix that I want to use [this](http://rusvectores.org/static/models/rusvectores4/fasttext/araneum_none_fasttextskipgram_300_5_2018.tgz) pretrained FastText model to obtain vectors. How can I do that?

**Comment by honnibal** [comment]
spaCy learns vectors during training, by back-propagating loss to the tokens. It does this even if you're providing pre-trained vectors (the learned vectors are kept separate from the pre-trained ones).

To use the pre-trained vectors, you can use the `spacy init-model` command, with the `--vectors` argument. You just need to give it a path to the zipfile with the FastText vectors. Then you pass the resulting vocab file to the `spacy train` command, using the `--vocab` argument.
